tools/ci/nightly/extract-release-notes.js

```diff
@@ -0,0 +1,63 @@
+const fs = require("fs");
+
+const inputPath = process.argv[2];
+const outputPath = process.argv[3];
+const version = process.env.DIST_VERSION;
+
+console.log("Extracting release notes from " + inputPath + " to " + outputPath);
+
+/** Returns the part of the text until the second top-level heading (exclusive)
+ * in Markdown formatting.
+ */
+function cutFirstSection(content) {
+  const nightlySectionRegex = /^# Enso Next$/gm;
+  function findNightlySectionStart(text) {
+    return text.search(nightlySectionRegex);
+  }
+  const regularSectionRegex = /^# Enso .*? \(\d+-\d+-\d+\)$/gm;
```

Comment:
You can make this more specific. It's always going to be ISO date format, so it's 4 digits then 2 then 2.

Reply:
Hmmm, what happens in year 10000? :smile: